dolphinscheduler-spi/src/main/java/org/apache/dolphinscheduler/spi/enums/DbType.java

```diff
@@ -88,7 +88,7 @@ public String getDescp() {
     }
 
     public boolean isHive() {
-        return this == DbType.HIVE;
+        return this == DbType.HIVE || this == DbType.KYUUBI;
```

Comment:
May I ask why KYUUBI is also true in `isHive()`

Reply:
KYUUBI use spark engine to query hive table by default, using HiveReader is better.